container/dockershell.sh

```diff
@@ -1,32 +1,55 @@
-#!/bin/bash -x
+#!/bin/bash
 
 # path within container
 SETUP_PATH="/root/db2eventstore-IoT-Analytics/container/setup"
 USER_VOLUME=${HOME}/eventstore_demo_volume
 
+# Deployment type constants, default to cp4d
+declare -r deployTypeCp4d="cp4d"
+declare -r deployTypeDsx="dsx"
```

Comment:
call this WSL, it has not been DSX for a while now :-)

Reply:
I know but I really find it causes confusion between WSL add-on's in cp4d VS what we're referring too here !  I'll change it though.